portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*global var which holds the number of positive (index 0), neutral (index 1), and negative (index 2) comments */
+var commentEmotions = new Array(0,0,0);
```

Comment:
Does this have to be a global var? Global vars are often discouraged unless necessary as it is hard to track mutations that happen to global vars.

Reply:
+1 looks like you don't need to have this as a global variable.

Reply:
I made it a global var because I wanted to have one instance of that variable and update it numerous times when the post functions were called